lib/position.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Position
+  $count = 0
+  attr_accessor :get_position
+
+  def self.get_position_num
+    while $board.include?(' ')
+      @get_position = gets.strip.to_i
+      return @get_position if (1..9).include?(@get_position)
+
+      puts 'Error, invaild input!!! Choose a Number between 1 - 9'
```

Comment:
No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))